apps/categories2/routes.coffee

```diff
@@ -1,2 +1,14 @@
+Q = require 'bluebird-q'
+Genes = require '../../collections/genes'
+
 @index = (req, res) ->
-	res.render 'index'
+  genes = new Genes
+
+  Q.all([
+    genes.fetchUntilEndInParallel(cache: true, data: published: true, sort: 'name')
```

Comment:
👌 since this is a new app, I'll go ahead and add my obligatory encouragement to try [metaphysics](https://github.com/artsy/force/blob/master/lib/metaphysics.coffee) if this fetching/orchestration logic get's more complicated. Noting that 👍 to `cache: true` here given it's fetching all of the genes.